### PR TITLE
Support unicode in turbodbc_arrow; Add a conda+Python 3.7 build to the matrix

### LINKED ISSUE #128: Build Arrow Support on Windows
Currently we only build and test it on Linux and macOS

### LINKED ISSUE #165: executemanycolumns cause InterfaceError  UTF-16 Strings are not supported yet when use pyarrow table as parameter
Trying 
cursor.executemanycolumns("INSERT INTO my_table VALUES (?, ?,?, ?, ?, ?, ?, ?,?)", table)

My_table looks like:
image

 I got the following error:
  File "/opt/anaconda/lib/python3.6/site-packages/turbodbc/exceptions.py", line 54, in wrapper
    raise InterfaceError(str(e))
turbodbc.exceptions.InterfaceError: UTF-16 Strings are not supported yet

___________
turbodbc: 2.6.0
Python 3.6
anaconda
MS SQL server
running under linux opensuse

### LINKED ISSUE #166: Geting garbled up characters (e.g., u'\xe2\x99\xa5') when options = make_options(prefer_unicode=True) on MS SQL server
Getting query results looks like:
'n\x00v\x00a\x00r\x00c\x00h\x00a\x00r\x00'

### PR DESCRIPTION
**Pull request by xhochy**
This is a combination of 2 PRs with an additional commit to test MS SQL on Linux to get to a green coverage for the Arrow changes.

Fixes #194
Fixes #190
Fixes #128
Fixes #166
Fixes #165

### CODE REVIEW

**Comment by xhochy** [comment]
Rebased and build passed again.

**Comment by xhochy** [comment]
@MathMagique any chances you can have a look at this? CI is happy with it ;)

**Comment by MathMagique** [comment]
Actually, there is a good chance, because I have one week of leave time next week :-D. Perhaps I'll even manage today

File: appveyor.yml [hunk 1]
```diff
@@ -35,7 +35,7 @@ install:
 
     SET PATH=%PYTHON%;%PYTHON%\\Scripts;%PATH%
 
-    pip install numpy pytest pytest-cov mock
+    pip install numpy pytest pytest-cov mock "pyarrow>=0.11,<0.12" pandas
```
Comment (MathMagique) [inline appveyor.yml]: This is odd. You install pyarrow 0.12 under Linux, and also the current master requires pyarrow 0.12+.

Comment (xhochy) [reply appveyor.yml]: This was an artefact from rebasing. This branch is older than the pyarrow update; fixed.

**Comment by MathMagique** [comment]
A thousand thanks to you, @xhochy - sorry for the long wait

**Pull request merged**